testutil/keeper/keepers_init.go

```diff
@@ -21,6 +21,8 @@ import (
 	"github.com/lavanet/lava/x/pairing"
```

Comment:
change the PR name so it'll include the JIRA ticket ID and be more indicative that it relates to projects